src/arch/arm/object/smmu.c

```diff
@@ -272,7 +272,7 @@ exception_t decodeARMCBInvocation(word_t label, unsigned int length, cptr_t cptr
         cb = cap_cb_cap_get_capCB(cap);
         cbSlot = smmuStateCBNode + cb;
         setThreadState(NODE_STATE(ksCurThread), ThreadState_Restart);
-        smmu_tlb_invalidate_cb(cb, cap_vtable_root_get_mappedASID(cbSlot->cap));
+        smmu_tlb_invalidate_cb(cb, cap_vspace_cap_get_capMappedASID(cbSlot->cap));
```

Comment:
so this means smmuv2 + aarch64 only?

Reply:
The change applies to all aarch64 configurations.  These changes here are because the cap_vtable_root_get_mappedASID CPP macro was removed as it's not needed anymore (https://github.com/seL4/seL4/pull/801/files#diff-6916e963c01ff1638d47ccd93c325cb39419e814a584fc0b59e966958dbda834L46-L115)